tests/unit/tf/examples/test_06_advanced_own_architecture.py

```diff
@@ -27,10 +27,5 @@ def test_example_06_defining_own_architecture(tb):
     )
     tb.execute()
     metrics = tb.ref("metrics")
-    assert sorted(list(metrics.keys())) == [
-        "loss",
-        "rating_binary/binary_classification_task/auc",
-        "regularization_loss",
```

Comment:
Should probably check that if there's a layer with a regularization loss it gets propagated